css/index.scss

```diff
@@ -1,4 +1,6 @@
 :root {
+  color-scheme: light dark;
```

Comment:
Courtesy of `color-theme`

![begif](https://user-images.githubusercontent.com/1402241/140823149-33328b87-df42-4dfe-9c14-907683b6063e.gif)